docs/introduction/sla.md

```diff
@@ -0,0 +1,222 @@
+customer, Rahti Container Cloud Service and the service provider, CSC - IT
+Center for Science Ltd., to cover the provision and support of the service as
+described hereafter. Amendments, comments and suggestions must be addressed
+using the communication channels defined in the section [Communication,
+reporting and escalation](#communication-reporting-and-escalation). The
+service provider retains right to introduce changes to the infrastructure. If
+the Customer does not accept the changes, this service subscription can be
+terminated.
+
+## Scope and description of the service
+
+The Rahti container cloud service is a cloud computing service. The Rahti
+service is based on OKD, which is a distribution of Kubernetes. The Rahti
+service allows its customers or users to run their own applications on top of
+it, often implemented as containers. The Rahti service provides, but is not
+necessarily limited to, these resources: 
+
+ * Hosting of container instances, and groups of container instances
+ * Storage services 
+ * Virtual networks for connecting container instances
+ * Load balancing of traffic to user workloads
+ * Features for replication, rolling updates, auto-recovery and auto-scaling of user workloads
+ * Basic default domain name and TLS for hosted web applications (under rahtiapp.fi)
+
+Customers can manage their resources using a web interface accessible through a
+web browser and through a set of APIs which allow programmatic management of
+resources. In order to access and use the service the customer must have a CSC
+user account.
+
+The user's applications are separated from other users’ applications from a
+network, storage and computational view.
+
+The Rahti service accounts for service usage based on the resources used. Up to
+date resource cost information is found in the [Rahti documentation](https://rahti.csc.fi)
+
+The normal functioning of the service is defined as
+
+ * the ability to launch new user applications.
+ * the availablility and integrity of the storage services.
+ * the functioning of the network, and the accessibility of user applications.
+ * the functioning of replication, auto-recovery and auto-scaling of user applications.
+
+This service is provided under User policy and terms of usage for the Rahti
+contariner cloud service hence by agreeing to this document the customer also
+agrees with CSC General Terms of Use and CSC’s Security Policy.
+
+## Service hours & exceptions
+
+The service is designed to run continuously. However, the following exceptions apply:
+
+Scheduled maintenances:
+
+ * Planned service breaks - Announced to the customer at least three weeks in advance
+ * Unplanned service breaks
+
+## Service components & dependencies
+
+The Rahti service is depends on the following services. The dependencies of these services are described in their respective service documentation.
+
+ * The cPouta service
+ * CSC user administration
+ * CSC Servicedesk for user support
```

Comment:
This is "CSC’s helpdesk" in the cPouta SLA, though the wording here might be better.